Initial commit for Spanish language for basic pages.

yeyeto2788 opened the pull request:
Hello guys! I'm doing the tranlations to Spanish for the website. I'm doing it step by step.

I have left some links to the `en` language since I haven translated them. I'll be updating them as soon as I go deeper on it.

Regards.

P.D: I hope this is not comming out of the blue and if there is any issue just let me know.

carlosperate commented on es/discuss.html:
```diff
@@ -0,0 +1,33 @@
+i18n: es
+---
+
+<h1>Discutir sobre Mu</h1>
```
This is probably more of a personal view, but to me `Discutir` sounds more like "arguing" than "talking", perhaps we can use a synonum like "Hablar"?

carlosperate replied:
Was the word `forum` used in the original text? If that's the case `foro` sounds like a good translation to me.

About the word `discutir` in itself, I can't say there is a clear winner. `Hablar` is a good options, perhaps `conversar`? Whichever you prefer.

yeyeto2788 replied:
The `forum` word is not on the original text but for me sounds a bit better than `discutir`, in the other hand tough `conversar` sounds much better than `Hablar`.

I will change it to `conversar` in the mean time and if we can come up with a better one I'll change it again :)

carlosperate commented on es/discuss.html:
```diff
@@ -0,0 +1,33 @@
+donde la comunidad puede unirse.</p>
+
+<p>Las contribuciones son bienvenidas sin perjuicio de cualquier persona, independientemente de la edad,
+género, religión, raza o sexualidad. Si estás pensando, "pero no significan
```
Perhaps change `pero no significan yo` to `pero eso no me incluye`?

yeyeto2788 replied:
I actually do the changes from the computer adding this change.

carlosperate commented on es/discuss.html:
```diff
@@ -0,0 +1,33 @@
+<div class="alert alert-info" role="alert">
+<h4><strong>Código de conducta de la comunidad</strong></h4>
+<p>Esperamos que los colaboradores sigan el
+<a href="https://www.python.org/psf/codeofconduct/" target="_blank"><strong>Código de conducta</strong></a> de la fundación de Python Software (Python Software foundation).</p>
```
```suggestion
<a href="https://www.python.org/psf/codeofconduct/" target="_blank"><strong>Código de conducta</strong></a> de la fundación de Python Software (Python Software Foundation).</p>
```

carlosperate commented on es/download.md:
```diff
@@ -0,0 +1,95 @@
+si está en Raspbian (la versión de Linux para Raspberry Pi) puede instalar Mu como un paquete.
+
+Si eres un desarrollador, puedes encontrar el código fuente.
+[on GitHub](https://github.com/mu-editor/mu).
```
```suggestion
[en GitHub](https://github.com/mu-editor/mu).
```

carlosperate commented on es/download.md:
```diff
@@ -0,0 +1,95 @@
+ya (y debe usar el administrador de paquetes de su sistema operativo para instalarlo). Finalmente,
+si está en Raspbian (la versión de Linux para Raspberry Pi) puede instalar Mu como un paquete.
+
+Si eres un desarrollador, puedes encontrar el código fuente.
```
```suggestion
Si eres un desarrollador, puedes encontrar el código fuente
```

carlosperate commented on es/download.md:
```diff
@@ -0,0 +1,95 @@
+     debe asegurarse de agregar al grupo de permisos correcto (por lo general,
+     los grupos <code> dialout </code> o <code> uucp </code>). También hay que
+     asegurarse de que su distribución monta automáticamente dispositivos flash, o
+     Asegúrate de montarlos manualmente.</p>
```
```suggestion
     asegúrate de montarlos manualmente.</p>
```

yeyeto2788 commented on es/about.html:
```diff
@@ -0,0 +1,42 @@
+  <div class="col-lg-6">
+    <h4>Menos es Más.</h4>
+    <p>Mu tiene solo las características más esenciales, por lo que los usuarios no se sienten intimidados por una interfaz desconcertante.</p>
+    <h4>Construye el camino con la menos resistencia.</h4>
```
From my point of view `pisando` wouldn't fit on the phrase.

I would put something like: `Construye el camino con la menor resistencia`, take into account that there is also a typo (`camino con la menos resistencia`)

Note the `s` on the word `menos` it should also be changed to `menor`

What do you think?

carlosperate replied:
Yeah, I agree `pisando` doesn't quite work here.
I think I prefer `Camina el sendero...` in this case, it's probably closer to the original.

yeyeto2788 replied:
Ok, changing it right now then.

yeyeto2788 commented:
THANK YOU! @carlosperate

I highly appreciated the fact that you took the time to review this. I wasn't 100% sure of some things. I'll update the remaining comments in a single commit tomorrow and I'll try to add more things. 

If you have any other suggestion, please, do not hesitate on pointing them out.

Regards.

yeyeto2788 commented:
Hey @carlosperate, I have added another page and I have added some of the proposed solutions. There are 2 comments remaining that we might need to fix. If you know

* `es/discuss.html` regarding the translation of `Discuss`
* `es/about.html` about which is the best option for `Tread the Path of Least Resistance.`

yeyeto2788 commented on es/howto/bugs.md:
```diff
@@ -0,0 +1,61 @@
+
+Si crees que ha encontrado un error, simplemente haga clic en el botón de abajo
+(necesita tener una cuenta en GitHub) y luego haga clic en el botón verde "Nuevo problema"
+Se encuentra en la parte superior derecha de la página:
```
It has a typo, it should say:
`que se encuentra en la parte superior...`

yeyeto2788 commented:
@carlosperate, @ntoll  I think there is no pending reviews on this PR. I will be working again on translating next week. You can merge these changes if you guys want.

I'll add another PR for rest.

Regards.

yeyeto2788 commented:
@carlosperate, @ntoll by any chance, do you guys have images of mu in Spanish? I'm translating other pages but I guess that I'll need some pictures to be in Spanish.

Regards.

carlosperate commented:
If the local language in the OS is set to Spanish it should automatically
launch in that language.
Alternatively if you are running from source code you can edit the __init__
file and set the language manually to Spanish.

On Fri, 7 Dec 2018, 07:05 Juan Biondi <notifications@github.com wrote:

> @carlosperate <https://github.com/carlosperate>, @ntoll
> <https://github.com/ntoll> by any chance, do you guys have images of mu
> in Spanish? I'm translating other pages but I guess that I'll need some
> pictures to be in Spanish.
>
> Regards.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/mu-editor/mu-editor.github.io/pull/42#issuecomment-445142833>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AD_sTp4-zOPJdOZGVFg912-hlVXNpzXEks5u2hMzgaJpZM4YQHoz>
> .
>

yeyeto2788 commented:
@carlosperate great, I'll try to get those images out of running Mu locally on my environment. Hope it is ok.

If I get those images we'll need to define some things regarding directories and links. I hope there is no limit on hosting (Take into account that the more images the more space is needed)

Regards.

carlosperate commented:
About the hosting no need to worry, the website is hosted by GitHub pages, so it is free. However, is always important to think about the website loading time for the users, so we should always try to keep the image sizes reasonably low (no need to make the images tiny, but always compress them to jpg/png or similar).
